crates/test-utils/src/chain/deploy.rs

```diff
@@ -55,6 +54,38 @@ where
     Ok(u256)
 }
 
+pub fn calculate_time_stretch(rate: FixedPoint, position_duration: FixedPoint) -> FixedPoint {
```

Comment:
We use this in `agent0` and so it should not be moved without updating PRs elsewhere.
https://github.com/delvtech/agent0/blob/main/src/agent0/ethpy/hyperdrive/interface/_mock_contract.py#L25

Reply:
Wait I'm sorry, it looks like you did not delete the version in `hyperdrive-math/src/utils.rs`. Why did you duplicate the code here instead of using the imported version? Is this part of the circular dependency thing?

Reply:
yeah this would make another circular dependency :P.  This isn't great I know, but one problem at a time I think.

Reply:
I'll add a comment to the top of this one.